kernel/LinearAlgebra/MiddleSizeMatrix.cpp

```diff
@@ -775,25 +775,43 @@ void MiddleSizeMatrix::solveCholesky(LinearAlgebra::MiddleSizeMatrix& B) const {
     logger.assert_always(info == 0, "Error in solving using decomposition.");
 }
 
-void MiddleSizeMatrix::solveLowerTriangular(
-    LinearAlgebra::MiddleSizeMatrix& B) const {
+void MiddleSizeMatrix::solveLowerTriangular(LinearAlgebra::MiddleSizeMatrix& B,
+                                            bool left) const {
```

Comment:
what does `left` mean here? left <-> right or left/remaining?

Reply:
Improved this.